src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -1464,18 +1464,36 @@ public SwingActionSavePatientExamination() {
 
 		@Override
 		public void actionPerformed(ActionEvent actionEvent) {
-
-			try {
-				examinationBrowserManager.saveOrUpdate(patex);
-				modified = false;
-			} catch (OHServiceException ohServiceException) {
-				MessageDialog.showExceptions(ohServiceException);
+			double weight = patex.getPex_weight();
+			int height = patex.getPex_height();
+
+			if (weight != 0 && height != 0) {
+				if (weight == ExaminationParameters.WEIGHT_MIN || weight == ExaminationParameters.WEIGHT_MAX || height == ExaminationParameters.HEIGHT_MIN || height == ExaminationParameters.HEIGHT_MAX) {
+					int response = JOptionPane.showConfirmDialog(null, MessageBundle.getMessage("angal.patient.examination.defaultvalue.msg"), "Confirm", JOptionPane.YES_NO_OPTION);
+					if (response == JOptionPane.YES_OPTION) {
+						savePatientExamaination();
+					}
+				} else {
+					savePatientExamaination();
+				}
+			} else {
+				JOptionPane.showMessageDialog(null, MessageBundle.getMessage("angal.patient.examination.nonzero.msg"), "Error", JOptionPane.ERROR_MESSAGE);
```

Comment:
Here is another case where there is a helper method in the code base:  `MessageDialog.error()`   Once again it takes care of the localizing the string "Error" for you.